routes/feed.js

```diff
@@ -29,19 +36,19 @@ router.use(function(req, res, next) {
 router.get('/:feed', function(req, res, next) {
   var changeNumber;
   if (req.query.afterChangeNumber === undefined && req.query.afterTimestamp === undefined) {
-    changeNumber = moment().day(-14).unix();
+    changeNumber = moment().add(env.TIMESTAMP_START_DAY_OFFSET, 'days').unix();
   } else {
     changeNumber = parseInt(req.query.afterChangeNumber) || parseInt(req.query.afterTimestamp);
     if (isNaN(changeNumber)) {
-      next(new Error("Invalid afterChangeNumber or afterTimestamp"));
+      return next(new Error("Invalid afterChangeNumber or afterTimestamp"));
```

Comment:
This was a bug. It would have continued to process the function after giving the error to the callback